src/+datools/+statistical/+ensemble/EnF.m

```diff
@@ -1,35 +1,39 @@
 classdef EnF < handle
+    % This is the base class for all statistical methods
+    % Derive from this class and overload methods/functions as required
+    % Deriving from handle base class allows an object of this class to be
+    % passed by reference.
 
     properties
-        Model % ODE
+        Model % ODE type
```

Comment:
why is this "ODE type"? That is a model class wrapper around a model. It does not even technically have to be an ODE

Reply:
Does this sound good?
"type of ODE solver (ode45/Runge Kutta) and the model (eg: Lorenz63)"

Reply:
Not really. Honestly, make it reference the Model type. We support models that are not ODE based by the way, so that is also wrong what you have now.